src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__wrap {
+    display: flex;
+
+    &--4 :nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
You don't need additional class such as `__wrap`. It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars

Reply:
I don't understand what I should do